### ISSUE TITLE
Python version problem(fix provided),   [AttributeError: 'module' object has no attribute 'quote']

### ISSUE THREAD

**Issue by matrix1001** [pwndbg/pwndbg#388]
# Description
just like this
```python
pwndbg> start

Exception occured: entry: 'module' object has no attribute 'quote' (<type 'exceptions.AttributeError'>)
```
problem file
`pwndbg/commands/start.py`
problem code(at the bottom of the file)
```python
run = 'run ' + ' '.join(map(shlex.quote, a))
```
this is due to python version problem. in py3, it could be ok. but in py2, `shlex` does not have `quote`.
# Solution
just make some changes
```python
@pwndbg.commands.Command
@pwndbg.commands.OnlyWithFile
def entry(*a):
    """
    Set a breakpoint at the first instruction executed in
    the target binary.
    """
    global break_on_first_instruction
    break_on_first_instruction = True
    try:
        from shlex import quote
    except ImportError:
        from pipes import quote
    run = 'run ' + ' '.join(map(quote, a))
    gdb.execute(run, from_tty=False)
```
# Suggestion
Hope u will fix it as soon as possible, with better solution mabe.

**Comment by disconnect3d** [comment]
Hey, thx for reporting this bug!

The fix is fine, we just have to put it at the top of the file (it is not really cool to have imports in between code).

**Comment by disconnect3d** [comment]
Fixed in PR #389
